app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,22 +1,75 @@
-import type { ChangeEvent } from 'react';
+import { PageHeading } from '@18f/identity-components';
 import { t } from '@18f/identity-i18n';
+import { formatHTML } from '@18f/identity-react-i18n';
 import { FormStepsButton } from '@18f/identity-form-steps';
 import type { FormStepComponentProps } from '@18f/identity-form-steps';
-import type { VerifyFlowValues } from '../../verify-flow';
+import type { VerifyFlowValues } from '../..';
+import { Accordion } from '@18f/identity-components';
+import parsePhoneNumber from 'libphonenumber-js';
```

Comment:
Via [Slack](https://gsa-tts.slack.com/archives/C03DCUECFH7/p1652361313760439?thread_ts=1652300619.363169&cid=C03DCUECFH7) and [previous PR](https://github.com/18F/identity-idp/pull/6306#discussion_r865828380), we should cherry-pick the methods we're using to avoid bloating the size of the bundle.

```js
import { parse, format } from 'libphonenumber-js';
const phoneNumber = parse(`+1${pii?.phone}`);
const formatted = format(phoneNumber, 'NATIONAL');
```